app/src/main/java/nerd/tuxmobil/fahrplan/congress/sharing/SimpleSessionFormat.java

```diff
@@ -16,41 +16,49 @@
     private static final String COMMA = ",";
     private static final String SPACE = " ";
     private static final String HORIZONTAL_DIVIDERS = "---";
+    private static final String ROUND_BRACKET_OPENING = "(";
+    private static final String ROUND_BRACKET_CLOSING = ")";
 
 
     @NonNull
-    public static String format(@NonNull Session session) {
+    public static String format(@NonNull Session session, @NonNull String timeZoneName) {
         StringBuilder builder = new StringBuilder();
-        appendSession(builder, session);
+        appendSession(builder, session, timeZoneName);
         return builder.toString();
     }
 
     @Nullable
-    public static String format(@NonNull List<Session> sessions) {
+    public static String format(@NonNull List<Session> sessions, @NonNull String timeZoneName) {
         if (sessions.isEmpty()) {
             return null;
         }
         int sessionsSize = sessions.size();
         if (sessionsSize == 1) {
-            return format(sessions.get(0));
+            return format(sessions.get(0), timeZoneName);
         }
         StringBuilder builder = new StringBuilder();
         for (int i = 0; i < sessionsSize; ++i) {
             Session session = sessions.get(i);
-            appendSession(builder, session);
+            appendSession(builder, session, timeZoneName);
             if (i < sessionsSize - 1) {
                 appendDivider(builder);
             }
         }
         return builder.toString();
     }
 
-    private static void appendSession(StringBuilder builder, Session session) {
+    private static void appendSession(StringBuilder builder, Session session, String timeZoneName) {
         long startTime = session.getStartTimeMilliseconds();
         String shareableStartTime = DateFormatter.newInstance().getFormattedShareable(startTime);
```

Comment:
Since you now pass the time zone you should also use it when formatting the date (instead of using the device time zone).

Reply:
:heavy_check_mark: Addressed in 3a9baf0aa7e54e7ff22e3525d0fed7bacce8e77e.